ARM-wvd-templates/Create and provision WVD host pool/README.md

```diff
@@ -0,0 +1,112 @@
+Click the button below to deploy:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreate%20and%20provision%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreate%20and%20provision%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+# ARM Template to Create and provision new Windows Virtual Desktop hostpool

```

Comment:
I some of the info from this point on doesn't apply to CreateHostpoolTemplate.json , for example the "Authentication to Windows Virtual Desktop" section.

Reply:
ok i fixed it up a bit i think everything else is relevant